packages/components/src/components/flipcard/flipcard.stories.ts

```diff
@@ -0,0 +1,275 @@
+            },
+            {
+              type: 'attribute',
+              name: 'frontVariant',
+              value: 'gradient-dark-top'
+            },
+            {
+              type: 'attribute',
+              name: 'backVariant',
+              value: 'gradient-dark-bottom'
+            }
+          ]
+        });
+      })}
+    `;
+  }
+};
+
+/**
+ * Use the `base`, `front`, `back`, `frontSlotContainer`, `backSlotContainer`, `mediaFront`, `mediaBack`, `frontSecondaryGradient` and `backSecondaryGradient` parts to style the flipcard.
+ */
+export const Parts = {
+  parameters: {
+    controls: {
+      exclude: [
+        'base',
+        'front',
+        'back',
+        'frontSlotContainer',
+        'backSlotContainer',
+        'mediaFront',
+        'mediaBack',
+        'frontSecondaryGradient',
+        'backSecondaryGradient'
+      ]
+    }
+  },
+  render: (args: any) => {
+    return generateTemplate({
+      axis: {
+        y: {
+          type: 'template',
+          name: 'sd-flipcard::part(...){outline: solid 2px red}',
+          values: [
+            'base',
+            'front',
+            'back',
+            'frontSlotContainer',
+            'backSlotContainer',
+            'mediaFront',
+            'mediaBack',
+            'frontSecondaryGradient',
+            'backSecondaryGradient'
+          ].map(part => {
+            return {
+              title: part,
+              value: `<style>#part-${part} sd-flipcard::part(${part}){outline: solid 2px red; outline-offset: -2px;}</style><div id="part-${part}">%TEMPLATE%</div>`
+            };
+          })
+        }
+      },
+      args,
+      constants: [
+        {
+          type: 'template',
+          name: 'style',
+          value: '<div style="margin-bottom: 40px; width: 300px">%TEMPLATE%</div>'
+        },
+        {
+          type: 'attribute',
+          name: 'frontVariant',
+          value: 'gradient-dark-top'
+        },
+        {
+          type: 'attribute',
+          name: 'darkVariant',
+          value: 'gradient-dark-bottom'
+        }
+      ]
+    });
+  }
+};
+
+/**
+ * `sd-flipcard` is fully accessibile via keyboard.
+ */
+
+export const Mouseless = {
+  render: (args: any) => {
+    return html`<div class="mouseless" style="width:300px">${generateTemplate({ args })}</div>`;
+  },
+
+  play: async ({ canvasElement }: { canvasElement: HTMLUnknownElement }) => {
+    const el = canvasElement.querySelector('.mouseless sd-flipcard');
+
+    await waitUntil(() => el?.shadowRoot?.querySelector('.flip-card__side--front'));
+
+    // We have to catch the event as otherwise Storybook will break
+    await userEvent.type(el!.shadowRoot!.querySelector('.flip-card__side--front')!, '{return}', {
+      pointerEventsCheck: 0
+    });
+  }
+};
+
+/**
+ * Here is a sample of the `sd-flipcard` with custom content in the `front` and `back` slots.
+ */
+
+export const Sample = {
```

Comment:
this is still WIP right? If not, we need to discuss the UX here.
As the flipcard flips on hover, you never will be able to click the link on the front... This does not make sense at all.

@coraliefeil was the `flip-on-hover` actually a result from the refinement? I see this as a major issue especially as I have in mind, automatic behavior needs to be stopped if necessary (WCAG requirement?!?). Users need to decied on wich speed they want the content to change (not the duration but the point where it should start, like autoplay stuff with media).

Reply:
As this is user controlled, I believe it's fine. But we should definitely consider to turn off the animation here or make a simpler one instead, like a simple dissolve for people with reduced motion.

Reply:
Definitely a WIP @karlbaumhauer . This story did need some discussion. 

Mainly, I think it is about setting the flip-on-hover behaviours as a default. As this is clearly problematic anywhere where a user interaction is needed. I don't think any thing should be triggered by a hover when there are links/buttons present. 

I am not sure how the simpler animation would help here as users would still have issues interacting with the first page. (Maybe I am missing something here @mariohamann ) I see two possibilities here: 

Option 1: We remove the flip-on-hover behaviour and introduce an attribute to add this behaviour should users want it. So by default only clicking would work. 

Option 2: If we want to keep the flip-on-hover behaviour, we can add something like an 'interactive' attribute that disables flipping for hover.

I am leaning towards option 2 now. But what do you think @karlbaumhauer @mariohamann @coraliefeil ?

Reply:
What was the solution here as it is marked as "resolved"? I still think the hover behavior is quite a pain as we still have to make it by click for mobile/touch devices.

Reply:
forget it, saw the new comment above.